mailchimp_auth/views.py

```diff
@@ -54,40 +54,41 @@ class SignUpForm(JSONFormResponseMixin, FormView):
     def form_valid(self, form):
         email = form.cleaned_data['email']
 
-        token = form.data['g-recaptcha-response']
+        token = form.data.get('g-recaptcha-response', False)
 
-        try:
-            score = self._get_captcha_score(token)
-        except ValidationError:
-            raise
-        except requests.exceptions.ContentDecodingError:
-            raise ValidationError('Could not get reCAPTCHA score')
-        else:
-            user_is_a_bot = score < getattr(settings, 'GOOGLE_CAPTCHA_BOT_THRESHOLD', 0.1)
+        if token:
+            try:
+                score = self._get_captcha_score(token)
+            except ValidationError:
```

Comment:
not your code, but we let's also remove this line

```python
except ValdiationError:
     raise
```

is going to do the same thing as omitting those lines